.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Support Question
+    url: https://discourse.laminas.dev/c/questions/5
+    about: 'This repository is only for reporting bugs and suggesting features. Support is provided via forums:'
+  - name: 💡 Feedback Wanted
+    url: https://discourse.laminas.dev
+    about: 'Discuss and get feedback on ideas on our forums:'
```

Comment:
Should we extend the sentence to clarify what ideas are meant? Like:

> Discuss and get feedback on ideas for improvements or new features on our forums:

Or would this only be restrictive?

Reply:
Generally, improvement and feature requests would end up as issues I think.
For this I had more broad feedback requests in mind, like those issues where users showcasing library they built on top of the component.